core/handler/http_jrpc_client.py

```diff
@@ -1,125 +1,125 @@
-        """
-
-        data = request.encode('utf8')
-        datalen = len(data)
-
-        url = self.ServiceURL
-
-        logger.debug('post request to %s with DATALEN=%d, DATA=<%s>',
-                     url, datalen, data)
-
-        try:
-            request = urllib.request.Request(
-                url, data,
-                {'Content-Type': 'application/json',
-                 'Content-Length': datalen})
-            opener = urllib.request.build_opener(self.ProxyHandler)
-            response = self._open_with_retries(
-                opener, request, retries)
-
-        except urllib.error.HTTPError as err:
-            logger.warn('operation failed with response: %s', err.code)
-            raise MessageException(
-                'operation failed with response: {0}'.format(err.code))
-
-        except urllib.error.URLError as err:
-            logger.warn('operation failed: %s', err.reason)
-            raise MessageException('operation failed: {0}'.format(err.reason))
-
-        except Exception as err:
-            logger.exception('no response from server: %s', str(err))
-            raise MessageException('no response from server: {0}'.format(err))
-
-        content = response.read()
-        headers = response.info()
-        response.close()
-
-        encoding = headers.get('Content-Type')
-        if encoding != 'application/json':
-            logger.info('server responds with message %s of type %s',
-                        content, encoding)
-            return None
-
-        # Attempt to decode the content if it is not already a string
-        try:
-            content = content.decode('utf-8')
-        except AttributeError:
-            pass
-        value = json.loads(content)
-        return value
-
-    def _open_with_retries(self, opener, request, retries):
-        """
-        Function to retry opening a given url/request if URLError is
-        encountered. URLError for request would encompass HTTPError
-        as well as Timeout.
-
-        Parameters:
-            @param opener - An instance of OpenerDiretor
-            @param request - Request to be sent to the url
-            @param retries - Number of attempts to open
-        Returns:
-            @returns response - Response received
-        """
-        count = 0
-        while count < retries:
-            try:
-                return opener.open(request, timeout=10)
-            except urllib.error.URLError as err:
-                logger.error("Connection error - %s", err.reason)
-                time.sleep(10)
-            except Exception as err:
-                raise err
-            # Increment counter for each handled Exception
-            count += 1
-            if count < retries:
-                logger.info("Will retry to connect.")
-        # Make a final call after retries are exhausted
-        return opener.open(request, timeout=10)
+# Copyright 2019 Intel Corporation

```

Comment:
This file is moved to different folder, but why the diff is showing entire file has modified?